### PR TITLE
[IMP] stock_move_quick_lot: Allow to assign existing lots in detailed operations for pickings with create lots option enable.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT30997

ping @chienandalu @carlosdauden

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Tested! It's weird Odoo just don't consider this typical use case

File: stock_move_quick_lot/models/__init__.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 # from . import stock_move_line
```
Comment (chienandalu) [inline stock_move_quick_lot/models/__init__.py]: Just uncomment here or delete the line

**Comment by sergio-teruel** [comment]
@chienandalu Changes done!!

**Comment by chienandalu** [comment]
:+1:

**Comment by sergio-teruel** [comment]
I will port the changes to v13

**Pull request merged**